app/src/androidTest/java/com/github/polypoly/app/base/game/GameTest.kt

```diff
@@ -0,0 +1,93 @@
+package com.github.polypoly.app.base.game
+
+import com.github.polypoly.app.base.game.rules_and_lobby.GameLobby
+import com.github.polypoly.app.base.game.rules_and_lobby.GameMode
+import com.github.polypoly.app.base.game.rules_and_lobby.GameRules
+import com.github.polypoly.app.base.user.Skin
+import com.github.polypoly.app.base.user.Stats
+import com.github.polypoly.app.base.user.User
+import com.github.polypoly.app.map.LocationRepository
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import kotlin.time.Duration.Companion.hours
+
+@RunWith(JUnit4::class)
+class GameTest {
+
+    private val emptySkin = Skin(0,0,0)
+    private val zeroStats = Stats(0,0,0,0,0)
+    private val testUser1 = User(42042042, "test_user1", "", emptySkin, zeroStats, listOf(), mutableListOf())
+    private val testUser2 = User(42042043, "test_user2", "", emptySkin, zeroStats, listOf(), mutableListOf())
+    private val testUser3 = User(42042044, "test_user3", "", emptySkin, zeroStats, listOf(), mutableListOf())
+    private val testUser4 = User(42042045, "test_user4", "", emptySkin, zeroStats, listOf(), mutableListOf())
+    private val testUser5 = User(42042046, "test_user5", "", emptySkin, zeroStats, listOf(), mutableListOf())
+    private val testUser6 = User(42042047, "test_user6", "", emptySkin, zeroStats, listOf(), mutableListOf())
+    private val testDuration = 2
+    private val gameRules = GameRules(GameMode.RICHEST_PLAYER, 3, 7,
+        testDuration, 10, LocationRepository.getZones(), 200)
+    private val gameLobby = GameLobby(testUser1, gameRules, "test_game", "123456", false)
+
+    @Before
+    fun startIntents() {
+        gameLobby.addUser(testUser2)
+        gameLobby.addUser(testUser3)
+        gameLobby.addUser(testUser4)
+        gameLobby.addUser(testUser5)
+        gameLobby.addUser(testUser6)
```

Comment:
You can inherit from the newly created PolyPolyTest which does this for you.

Reply:
I want to create my own game specific to my test